### PR TITLE
Add makefile support for ovn

### PR DESCRIPTION
**Pull request by vthapar**
* Adds `using=ovn` flag
* Adds `cluster_settings.ovn` file
* Handles dummy code to handle `ovn` cni in clusters.sh

Fixes: #335

Signed-off-by: Vishal Thapar <5137689+vthapar@users.noreply.github.com>

### CODE REVIEW

File: scripts/shared/clusters.sh [hunk 1]
```diff
@@ -82,6 +83,9 @@ function create_kind_cluster() {
     fi
 
     echo "Creating KIND cluster..."
+    if [[ "${cluster_cni[$cluster]}" == "ovn" ]]; then
+        deploy_kind_ovn
```
Comment (mkolesnik) [inline scripts/shared/clusters.sh:87]:
```suggestion
        deploy_kind_ovn
        return
```

File: Makefile [hunk 2]
```diff
@@ -2,13 +2,17 @@ ifneq (,$(DAPPER_HOST_ARCH))
 
 # Running in Dapper
 
+include Makefile.inc
+
+ifneq (,$(filter ovn,$(_using)))
+CLUSTER_SETTINGS_FLAG = --cluster_settings $(DAPPER_SOURCE)/scripts/cluster_settings.ovn
+else
 CLUSTER_SETTINGS_FLAG = --cluster_settings $(DAPPER_SOURCE)/scripts/cluster_settings
+endif
```
Comment (mkolesnik) [inline Makefile:11]:
```suggestion
endif

```

**Review by mkolesnik — CHANGES_REQUESTED** [review]
Looks good so far, a couple of small suggestions

File: scripts/shared/clusters.sh [hunk 3]
```diff
@@ -82,6 +83,10 @@ function create_kind_cluster() {
...
+        return
+    fi
```
Comment (mkolesnik) [inline scripts/shared/clusters.sh:89]:
```suggestion
    fi

```

Sorry missed this nit the first time

**Review by mkolesnik — APPROVED** [review]
Looks good except one small nit but not blocking for merge

Don't forget to squash when merging, please :)

**Review by skitt — APPROVED** [review]

**Review by mangelajo — APPROVED** [review]
Looks good to me, we will need an additional PR to label any of the worker nodes as gateway.

**Comment by dfarrell07** [comment]
This was backported in https://github.com/submariner-io/shipyard/pull/436.

**Pull request merged**